requirements/test.txt

```diff
@@ -247,7 +259,7 @@ snowballstemmer==2.2.0
     # via
     #   pydocstyle
     #   sphinx
-sphinx==4.5.0
+sphinx==5.0.0
```

Comment:
Some [incompatible changes](https://github.com/sphinx-doc/sphinx/blob/5.x/CHANGES#L43) but not related to the usage in this package.